pages/index.js

```diff
@@ -2,21 +2,48 @@ import AddCalories from "../components/AddCalories/AddCalories";
 import useCalorieStore from "../utils/useCalorieStore";
 import styled from "styled-components";
 import ConsumedList from "../components/ConsumedList/ConsumedList";
+import { useEffect } from "react";
 
 const StyledDiv = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
 `;
-
+const ListContainer = styled.div`
+  position: absolute;
+  top: 4rem;
+  width: 274px;
```

Comment:
It's seldomly a good idea to have fixed widths. Reconsider this please.

Reply:
I will figure this out in the final styling